lib/ret/account_context.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Ret.AccountContext do
```

Comment:
Since there are rules governing the use of things within a bounded context, I often start with a single, general-purpose context, then split if needed once I can see the patterns in module collaboration.

Reply:
Are you suggesting that this should just be `Ret.Context`?

Reply:
I’d probably just call it `Ret`

Reply:
If you’re partial to having `Context` in the name, I might go with `Ret.GeneralContext`.

Reply:
Went ahead and put the functions directly under the main `Ret` module. Though I wasn't sure what the test should be called. I went with `context_test.exs` and `Ret.ContextTest`.

Reply:
Elixir doesn’t care, but if you want it to match you can move it to `test/ret_test.exs` and name it `RetTest`.

Reply:
:+1: makes sense